### PR TITLE
[risk=low][RW-7670] Set ERA Required = false by default for institutions

### PR DESCRIPTION
**Pull request by jmthibault79**
Also do a bit of refactoring

No-ERA

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/main/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [x] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/utils/institutions.tsx [hunk 1]
```diff
@@ -135,23 +135,7 @@ export function getTierConfig(
   institution: Institution,
   accessTierShortName: string
 ): InstitutionTierConfig {
-  if (!institution.tierConfigs) {
```
Comment (jmthibault79) [inline ui/src/app/utils/institutions.tsx:138]: getTierConfigOrDefault() now handles this case (just above here)

File: ui/src/app/utils/institutions.tsx [hunk 2]
```diff
@@ -135,23 +135,7 @@ export function getTierConfig(
...
-    return defaultTierConfig(accessTierShortName);
-  }
-
-  return getTierConfigOrDefault(institution.tierConfigs, accessTierShortName);
-}
-
-export function getRegisteredTierConfig(
```
Comment (jmthibault79) [inline ui/src/app/utils/institutions.tsx:145]: these are no longer needed

File: ui/src/app/pages/admin/admin-institution-edit.tsx [hunk 3]
```diff
@@ -159,8 +158,8 @@ const EnableCtSwitch = (props: {
       data-test-id='controlled-enabled-switch'
       onChange={(v) => onChange(v.value)}
       checked={
-        getControlledTierConfig(institution).membershipRequirement !==
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/admin-institution-edit.tsx:162]: removed tier-specific versions of this, since this was the only remaining reference

**Review by freemabd — APPROVED** [review]
LGTM

**Pull request merged**